pkg/health/health.go

```diff
@@ -0,0 +1,41 @@
+package health
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spiffe-helper/cmd/spiffe-helper/config"
+	"github.com/spiffe/spiffe-helper/pkg/sidecar"
+)
+
+func StartHealthServer(hclConfig *config.Config, log logrus.FieldLogger, sidecar *sidecar.Sidecar) error {
```

Comment:
Rather than pass the full config here can we just take in the health check config? Can you create a `Config` struct in this directory and have main populate and pass that in here?

Reply:
If we did, we'd also need to pass `DaemonMode` as a separate argument.

Reply:
It can just be part of the same config struct

Reply:
So create a new HealthServerConfig struct?